force_bdss/events/mco_events.py

```diff
@@ -90,6 +90,16 @@ def from_json(cls, json_data):
         return cls(**data)
 
 
+class MCORuntimeEvent(BaseDriverEvent):
+    """ The base class for the MCO events fired during the workflow
+     execution. For example, if a DataSource is designed to notify
+     the Workflow or the UI, this event would be an MCORuntimeEvent.
```

Comment:
Can we make it clear why this class does not inherit from `UIEventMixin` and therefore will not be reported back to the `WfManager` by default? Otherwise the docstring here might mislead a developer

Reply:
That is true, thank you. I will find a better explanation for this case.